README.md

```diff
@@ -29,6 +35,9 @@ The Matchbox project contains:
 
 ## How it works
 
+![Connection](https://raw.githubusercontent.com/johanhelsing/matchbox/main/images/connection.light.excalidraw.svg#gh-light-mode-only)
+![Connection](https://raw.githubusercontent.com/johanhelsing/matchbox/main/images/connection.dark.excalidraw.svg#gh-dark-mode-only)
```

Comment:
We should probably still replace this with just one image. If worst comes to worst, if we can't find a color that works for both schemes, we can just add a background to it.

Reply:
Replaced it with a single image.